submissions/youhymuk/html-css-popup/index.html

```diff
@@ -0,0 +1,189 @@
+<!doctype html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport"
+        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Hooli</title>
+  <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="./styles/styles.css">
+</head>
+<body>
+<nav class="nav">
+  <ul class="toolbar">
+    <li class="toolbar-item">
+      <a class="mail link" href="https://mail.google.com/mail/u/0/?tab=wm&ogbl" about="_blank">
+        Mail
+      </a>
+    </li>
+    <li class="toolbar-item">
+      <a class="images link" href="https://www.google.com.ua/imghp?hl=en&tab=wi&ogbl" about="_blank">
+        Images
+      </a>
+    </li>
+    <li class="toolbar-item popup-wrapper">
+      <input id="popup-btn" class="popup-btn" type="checkbox">
+      <label for="popup-btn" class="popup-icon"></label>
+      <div class="popup">
+        <div class="popup-content">
+          <ul class="menu">
+            <li class="menu-item">
+              <a href="#">
+                <figure class="menu-item-icon">
```

Comment:
This is not a bug but as a possible option. If you write selectors like BEM, then it makes sense to do like this:
```suggestion
                <figure class="menu-item__icon">
```

Reply:
Already replaced, thank you for advice)